content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/fedex.rst

```diff
@@ -0,0 +1,165 @@
+=================
+FedEx integration
+=================
+
+Integrating a FedEx account with Odoo's *Inventory* app makes it possible to :doc:`calculate
```

Comment:
Since according to the new guidelines decided yesterday, we bold app names now
```suggestion
Integrating a FedEx account with Odoo's **Inventory** app makes it possible to :doc:`calculate
```